pom.xml

```diff
@@ -86,6 +87,9 @@
     </ciManagement>
 
     <modules>
+<!--shouldn't this always be first?-->
```

Comment:
It doesn't matter. Maven figures out it needs to first compile core.

Reply:
this is new information for me.  it's definitely been a source of pain in the past